### PR TITLE
Fixed GameTime bug

### PR DESCRIPTION
**Pull request by glopezdiest**
### Description

`GameTime.restart()` now additional resets *GameTime._carla_time* and *GameTime._last_frame*.

This isn't creating any issues right now as CARLA's frame number isn't being reset upon loading the world. However, this will cause issues once 0.9.11 comes out, as the new sync mode does reset the frame number.

### Where has this been tested?

  * **Platform(s):** Ubuntu 18.04
  * **Python version(s):** 3.6
  * **Unreal Engine version(s):** 4.24
  * **CARLA version:** 0.9.10

---
This change is [Reviewable](https://reviewable.io/reviews/carla-simulator/scenario_runner/711)

### CODE REVIEW

**Comment by fabianoboril** [comment]
Can you update the branch to current master, and maybe add a changelog entry? Thanks :)

**Review by fabianoboril — CHANGES_REQUESTED** [review]
> Reviewed 1 of 1 files at r1, 1 of 1 files at r2.
> *[Reviewable](https://reviewable.io/reviews/carla-simulator/scenario_runner/711)* status: all files reviewed, 1 unresolved discussion (waiting on @glopezdiest)

___
*[Docs/CHANGELOG.md, line 29 at r2](https://reviewable.io/reviews/carla-simulator/scenario_runner/711#-MP57barArSZ-fXu34x-:-MP57barArSZ-fXu34x0:bxibm7d) ([raw file](https://github.com/carla-simulator/scenario_runner/blob/c19f294372e6e460db4c68026957e317de871312/Docs/CHANGELOG.md#L29)):*
> ```Markdown
> * Fixed bug causing the InTimeToArrivalToVehicle atomic to crash if one of the actors was a a static object
> * Fixed writing result files when using OpenSCENARIO under Windows (CARLA: prefix is removed from the filename)
> * Fixed bug causing the *GameTime* class at *srunner/scenariomanager/timer.py* to not correctly reset the frame number
> ```

I wouldn't call this a bug. Up to now, this was never an issue. So this is more a requirement for 0.9.11. Maybe rephrase this to "Updated GameTime calls ..."

**Review by glopezdiest — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/carla-simulator/scenario_runner/711)* status: 1 of 2 files reviewed, 1 unresolved discussion (waiting on @fabianoboril)

___
*[Docs/CHANGELOG.md, line 29 at r2](https://reviewable.io/reviews/carla-simulator/scenario_runner/711#-MP57barArSZ-fXu34x-:-MP59sn57NjaAX587i-j:b-896fix) ([raw file](https://github.com/carla-simulator/scenario_runner/blob/c19f294372e6e460db4c68026957e317de871312/Docs/CHANGELOG.md#L29)):*
<details><summary><i>Previously, fabianoboril (Fabian Oboril) wrote…</i></summary><blockquote>

I wouldn't call this a bug. Up to now, this was never an issue. So this is more a requirement for 0.9.11. Maybe rephrase this to "Updated GameTime calls ..."
</blockquote></details>

Done.

**Review by fabianoboril — APPROVED** [review]
> Reviewed 1 of 1 files at r3.
> *[Reviewable](https://reviewable.io/reviews/carla-simulator/scenario_runner/711)* status: :shipit: complete! all files reviewed, all discussions resolved
